### PR TITLE
make eventtrace::func_exit include elapsed time of this function and code cleanup.

### PR DESCRIPTION
**Pull request by majianpeng**

### CODE REVIEW

File: src/msg/async/AsyncConnection.cc [hunk 1]
```diff
@@ -432,9 +429,7 @@ void AsyncConnection::process()
 
       case STATE_OPEN_MESSAGE_HEADER:
         {
-#if defined(WITH_LTTNG) && defined(WITH_EVENTTRACE)
-          ltt_recv_stamp = ceph_clock_now();
-#endif
+          recv_stamp = ceph_clock_now();
```
Comment (tchaikov) [inline src/msg/async/AsyncConnection.cc]: this changes how we updates `recv_stamp`. hence not in the scope of "cleanup"? moreover, there is no need to update `recv_stamp` at this moment.

File: src/msg/async/AsyncConnection.cc [hunk 2]
```diff
@@ -2265,10 +2267,14 @@ ssize_t AsyncConnection::write_message(Message *m, bufferlist& bl, bool more)
   } else {
     ldout(async_msgr->cct, 10) << __func__ << " sending " << m << " continuely." << dendl;
   }
+
+#if defined(WITH_LTTNG) && defined(WITH_EVENTTRACE)
```
Comment (tchaikov) [inline src/msg/async/AsyncConnection.cc]: ditto.

File: src/common/EventTrace.cc [hunk 3]
```diff
@@ -73,7 +73,7 @@ EventTrace::EventTrace(CephContext *_ctx, const char *_file, const char *_func,
 {
   if (unlikely(!ctx)) 
     return;
-  last_ts = ceph_clock_now();
+  entry_ts = last_ts = ceph_clock_now();
```
Comment (majianpeng) [inline src/common/EventTrace.cc]: @rchagam. Hi Reddy, have you time to review this patch? Thanks!

**Comment by gregsfortytwo** [comment]
As this PR has conflicts and I'm not sure our lttng works or plans to work the same way as in the past, I am closing this PR.

**Pull request closed without merging**